tests/test_tacacs_global.rb

```diff
@@ -75,7 +75,9 @@ def test_tacacs_global
     global.encryption_key_set(nil, key)
     # Device encypts key - verify return value
     assert_equal(7, global.key_format)
-    assert_equal('"WAWY_NZB"', global.key)
+    key = 'WAWY_NZB'
+    key = "\"#{key}\"" unless Platform.image_version[/7\.3/]
```

Comment:
This is wrong. The getter should always add quotes and it's not for some reason. I'll debug and fix.

Reply:
Nevermind. My initial thoughts were correct and my changes are fine. No quotes for 7.3 images.

Ref: #519 #584